assets/scripts/lobby/sockets/sockets.js

```diff
@@ -570,6 +570,19 @@ socket.on('joinPassword', (roomId) => {
   })();
 });
 
+socket.on('timerUpdate', (timer) => {
+  const minutes = Math.floor(timer / 60).toString().padStart(2, '0');
+  const seconds = (timer % 60).toString().padStart(2, '0');
+  document.getElementById('timerDisplay').textContent = `Time remaining: ${minutes}:${seconds}`;
+});
+
+socket.on('timerFinished', () => {
+  document.getElementById('timerDisplay').textContent = 'Timer Finished!';
+});
+
+const duration = document.getElementById('timer-duration').value;
+socket.emit('SetTimer',,duration)
```

Comment:
This part looks a bit fishy. Should probably not be here, but in the game settings area instead!